### DISCUSSION TITLE
Localization of Auth component

### DISCUSSION THREAD

**Discussion by mellson** [supabase/supabase#6345] [Feature Requests]
It would be great if we could provide our own translations to the Auth component. Currently, it's only available in English, and it would be great to be able to provide our own versions of the texts.

**Comment by Tefiniaina** [comment]
@mellson to customize label, you can find it here https://supabase.com/docs/guides/auth/auth-helpers/auth-ui#custom-labels
So about the feedback message like error, can't find it.

**Reply by mellson** [reply]
Thanks, it's so great they added this to the new version of the Auth UI 👍🏻
